### PR TITLE
DTFS2-SCSS : Removed Inline CSS

### PR DESCRIPTION
**Pull request by abhi-markan**
## CSP
Due to no inline CSS permitted by `CSP`, all inline CSS have been removed and added to respective `SCSS` files.

### CODE REVIEW

**Review by Zainzzkk — APPROVED** [review]
LGTM

File: trade-finance-manager-ui/templates/case/facility/facility.njk [hunk 1]
```diff
@@ -26,7 +26,7 @@
 
 
   <div>
-    <h2 class="ukef-heading-grey" style="padding-top:90px">Facility {{ facility.ukefFacilityId }}</h2>
+    <h2 class="ukef-heading-grey" id="facility-ukef-heading-grey">Facility {{ facility.ukefFacilityId }}</h2>
```
Comment (IgnatG) [inline trade-finance-manager-ui/templates/case/facility/facility.njk:29]: do we not need that `padding-top: 90` in the css file?

Comment (IgnatG) [reply trade-finance-manager-ui/templates/case/facility/facility.njk:29]: ah. never mind. it's there

**Review by IgnatG — CHANGES_REQUESTED** [review]
You need to run `npm build` in every folder first

**Comment by abhi-markan** [comment]
> You need to run `npm build` in every folder first

Apologies, have build and pushed.

**Pull request merged**